.github/workflows/build.yml

```diff
@@ -14,20 +14,20 @@ jobs:
         Game: [Ares,TS,YR]
 
     steps:
-    - uses: actions/checkout@v3.1.0
+    - uses: actions/checkout@v4
       with:
         fetch-depth: 0
 
     - name: Setup .NET Core SDK
-      uses: actions/setup-dotnet@v3.0.3
+      uses: actions/setup-dotnet@v4
       with:
-        dotnet-version: '7.x.x'
+        dotnet-version: '8.x.x'
```

Comment:
Is this correct? I thought we were downgrading, or am I missing something

Reply:
Oh I see, we're working on v8 support. I missed that thread

Reply:
Even if we downgrade the client, we still need the latest toolchain from .NET 8 to make use of latest features. And we are adding .NET 8 support as well as .NET 4.8.